specifyweb/specify/datamodel.py

```diff
@@ -8202,7 +8204,107 @@
+            Field(name='text3', column='Text3', indexed=False, unique=False, required=False, type='java.lang.String', length=255)
+        ],
+        indexes=[
+
+        ],
+        relationships=[
+            Relationship(name='collection', type='many-to-one', required=False, relatedModelName='Collection', column='CollectionID'),
+            Relationship(name='taxonTreeDef', type='many-to-one', required=False, relatedModelName='TaxonTreeDef', column='TaxonTreeDefID'),
+            Relationship(name='createdByAgent', type='many-to-one', required=False, relatedModelName='Agent', column='CreatedByAgentID'),
+            Relationship(name='modifiedByAgent', type='many-to-one', required=False, relatedModelName='Agent', column='ModifiedByAgentID')
+        ],
+        fieldAliases=[
+
+        ]
+    ),
+    Table( # CollectionObjectGroup
+        sp7_only=True,
+        django_app='sp7_models',
+        classname='edu.ku.brc.specify.datamodel.CollectionObjectGroup',
+        table='collectionobjectgroup',
+        tableId=1016,
+        idColumn='CollectionObjectGroupID',
+        idFieldName='collectionObjectGroupId',
+        idField=IdField(name='collectionObjectGroupId', column='CollectionObjectGroupID', type='java.lang.Integer'),
+        fields=[
+            Field(name='name', column='Name', indexed=False, unique=False, required=True, type='java.lang.String', length=255),
+            Field(name='description', column='Description', indexed=False, unique=False, required=False, type='text'),
+            Field(name='igsn', column='IGSN', indexed=False, unique=False, required=False, type='java.lang.String', length=255),
+            Field(name='guid', column='GUID', indexed=False, unique=False, required=False, type='java.lang.String', length=255),
+            Field(name='number1', column='Number1', indexed=False, unique=False, required=False, type='java.lang.Integer'),
+            Field(name='version', column='Version', indexed=False, unique=False, required=False, type='java.lang.Integer'),
+            Field(name='timestampcreated', column='TimestampCreated', indexed=False, unique=False, required=True, type='java.sql.Timestamp'),
+            Field(name='timestampmodified', column='TimestampModified', indexed=False, unique=False, required=False, type='java.sql.Timestamp'),
+            Field(name='text1', column='Text1', indexed=False, unique=False, required=False, type='java.lang.String', length=255),
+            Field(name='text2', column='Text2', indexed=False, unique=False, required=False, type='java.lang.String', length=255),
+            Field(name='text3', column='Text3', indexed=False, unique=False, required=False, type='java.lang.String', length=255)
+        ],
+        indexes=[
+            Index(name='IGSN', column_names=['IGSN']),
+            Index(name='GUID', column_names=['GUID'])
+        ],
```

Comment:
I don't see these indexes materialized in the migration, see what's up with that.

Reply:
Ya, I didn't think so either, just put those there for now to see if anyone thought we might need them.